pkg/devfile/adapters/common/utils.go

```diff
@@ -92,7 +92,8 @@ type CommandNames struct {
 	AdapterName string
 }
 
-func isComponentSupported(component common.DevfileComponent) bool {
+// isComponentAContainer returns a bool if the component is a container
+func isComponentAContainer(component common.DevfileComponent) bool {
```

Comment:
Wouldn’t this be cleaner if it was a method on the component called `IsContainer`?

Reply:
Updated!